AirCasting/SessionViews/SessionCartViewModel/DefaultSyncingMeasurementsViewModel.swift

```diff
@@ -49,19 +49,37 @@ final class DefaultSyncingMeasurementsViewModel: SyncingMeasurementsViewModel {
                                   let longitude = measurement.longitude else { return CLLocationCoordinate2D(latitude: 200, longitude: 200) }
                             return CLLocationCoordinate2D(latitude: latitude, longitude: longitude)
                         }()
-                        guard let streamID = try? measurementStreamStorage.existingMeasurementStream(self.session.uuid, name: stream.sensorName) else {
-                            Log.info("failed to get existing streamID for synced measurements from session \(String(describing: self.session.name))")
-                            return }
-                        try? measurementStreamStorage.addMeasurement(Measurement(time: measurement.time, value: measurement.value, location: location), toStreamWithID: streamID)
+                        
+                        let sessionId = self.session.uuid!
+                        let sensorName = stream.sensorName
+                        let sessionName = self.session.name
+                        measurementStreamStorage.accessStorage { storage in
+                            do {
+                                guard let streamID = try? storage.existingMeasurementStream(sessionId, name: sensorName) else {
```

Comment:
we can omit `guard` and use a regular `try` since we're catching anyway

Reply:
We can't we need to pass nonoptional `streamID` to `addMeasurementValue`. Without guard `streamID` is optional. 
I can use `if let` instead of `guard let` 🤷🏻‍♀️